src/uu/sort/src/sort.rs

```diff
@@ -119,6 +121,7 @@ impl Default for GlobalSettings {
     fn default() -> GlobalSettings {
         GlobalSettings {
             mode: SortMode::Default,
+            debug: true,
```

Comment:
I would prefer to have `debug: false,` as default to explicit the default behavior
is it possible to change that?

Reply:
Sure, this was for initial testing.